src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs

```diff
@@ -125,7 +126,8 @@ public class AadIssuerValidator
                     {
                         if (IsV2Authority)
                         {
-                            AadIssuerV2 = validationParameters.Configuration.Issuer;
+                            BaseConfiguration openIdConnectConfig = validationParameters.ConfigurationManager.GetBaseConfigurationAsync(CancellationToken.None).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
@jennyf19 Had to mess around with your code when adding these changes; please let me know if something doesn't look right!

Reply:
actually @RojaEnnam has [changes there from a bug found when testing her config PR](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1780), which has the correct behavior.

Reply:
Ah, I see. Thanks for letting me know.
I'll incorporate some of her PR into this one; we'll need to coordinate how these PRs go in since they modify the same area.

Reply:
Ah, thanks for letting me know! I implemented some of her changes here.